src/schema/common.ts

```diff
@@ -167,8 +169,10 @@ export const Activity = zod.object({
     .object({
       large_image: zod.string().nullable(),
       large_text: zod.string().nullable(),
+      large_url: zod.string().url().optional().nullable(),
       small_image: zod.string().nullable(),
       small_text: zod.string().nullable(),
+      small_url: zod.string().url().optional().nullable(),
```

Comment:
unsure if we want only the _url fields to be optional but idk why some fields are only nullable and some fields are optional + nullable in the existing code.

Reply:
i did optional + nullable to align with what `details` and `state` had -- i imagine the reason why `large/small_image/text` are nullable but not optional is that the intention is `if (assets) { images & text should be defined }`